src/pushsource/_impl/model/ami.py

```diff
@@ -99,6 +99,21 @@ class AmiPushItem(VMIPushItem):
     )
     """Billing codes associated with this image."""
 
+    description = attr.ib(type=str, default=None, validator=optional_str)
+    """A description for the image"""
+
+    release_notes = attr.ib(type=str, default=None, validator=optional_str)
+    """"Release notes for AMI """
+
+    usage_instructions = attr.ib(type=str, default=None, validator=optional_str)
+    """Instructions for use of AMI"""
```

Comment:
Can this be a little more detailed too please?  Can you give an example of a string which would appear in this field?

Reply:
I have added more info in new commit.